### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by andrzejkukuryk**

### CODE REVIEW

File: src/data/personalLinks.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { ReactComponent as LinkedInIcon } from "../assets/linkedin.svg";
 import { ReactComponent as GitHubIcon } from "../assets/github.svg";
 import { ReactComponent as CvIcon } from "../assets/cv.svg";
+import i18n from "../i18n";
```
Comment (anna-morawska) [inline src/data/personalLinks.tsx:4]: do you use it somewhere?

File: src/data/projects.tsx [hunk 2]
```diff
@@ -8,6 +7,7 @@ import playiteasyMobile from "../graph/playiteasy_mobile.webp";
 import scalesDesktop from "../graph/scales_deskop.webp";
 import circleDesktop from "../graph/circle_desktop.webp";
 import circleMobile from "../graph/circle_mobile.webp";
+import i18n from "../i18n";
```
Comment (anna-morawska) [inline src/data/projects.tsx:10]: do you use it somewhere?

**Review by anna-morawska — APPROVED** [review]
nice 👍

File: src/data/personalLinks.tsx (hunk 1 above)
Comment (andrzejkukuryk) [reply src/data/personalLinks.tsx:4]: Deleted.

File: src/data/projects.tsx (hunk 2 above)
Comment (andrzejkukuryk) [reply src/data/projects.tsx:10]: Deleted.

**Pull request merged**